java/ql/lib/semmle/code/java/dataflow/ExternalFlowConfiguration.qll

```diff
@@ -28,17 +28,24 @@ private string childThreatModel(string group) {
 }
 
 /**
- * Holds if source models of the specified kind are
- * supported for the current query.
+ * Gets all kinds of threat models.
+ */
+private string getAllKinds() { sourceModel(_, _, _, _, _, _, _, result, _) }
+
+/**
+ * Gets the related source model kinds under the current threat model.
  */
 bindingset[kind]
-predicate supportedSourceModel(string kind) {
-  // expansive threat model includes all kinds
-  supportedThreatModel("expansive")
+string getSourceModelKinds(string kind) {
+  supportedThreatModel("standard") and
+  result = kind
```

Comment:
Not sure I understand this. It looks like this means: if `standard` is explicitly mentioned as a supported threat model, then this `kind` is supported (without checking any parents or children). Is that right? And if so is that what we want?

Reply:
My understanding of the EDR was that if "standard" was set as the threat model we want to use exactly the models that were originally intended (as a minimum).